### ISSUE TITLE
Some tests fail locally but not in GHA

### ISSUE THREAD

**Issue by etiennebacher** [easystats/bayestestR#597]
I have local failures for R CMD check in `test-blaavan.R` but these don't appear in GHA. Do you know why? Can you reproduce?

``` r
suppressPackageStartupMessages({
  library(bayestestR)
  library(blavaan)
  library(lavaan)
  library(cmdstanr)
  library(rstan)
  library(testthat)
})
data("PoliticalDemocracy", package = "lavaan")

model <- "
  # latent variable definitions
  dem60 =~ y1 + a*y2
  dem65 =~ y5 + a*y6

  # regressions
  dem65 ~ dem60

  # residual correlations
  y1 ~~ y5
"
invisible(capture.output(
  bfit <- blavaan::bsem(model,
                        data = PoliticalDemocracy,
                        n.chains = 1, burnin = 50, sample = 100
  )
))
#> as(<lgCMatrix>, "dgCMatrix") is deprecated since Matrix 1.5-0; do as(., "dMatrix") instead
#> Warning: The largest R-hat is 1.1, indicating chains have not mixed.
#> Running the chains for more iterations may help. See
#> https://mc-stan.org/misc/warnings.html#r-hat
#> Warning: Bulk Effective Samples Size (ESS) is too low, indicating posterior means and medians may be unreliable.
#> Running the chains for more iterations may help. See
#> https://mc-stan.org/misc/warnings.html#bulk-ess
#> Warning: Tail Effective Samples Size (ESS) is too low, indicating posterior variances and tail quantiles may be unreliable.
#> Running the chains for more iterations may help. See
#> https://mc-stan.org/misc/warnings.html#tail-ess
#> Warning: blavaan WARNING: As specified, the theta covariance matrix is neither diagonal nor unrestricted, so the actual prior might differ from the stated prior. See
#>  https://arxiv.org/abs/2301.08667

x <- point_estimate(bfit, centrality = "all", dispersion = TRUE)
expect_true(all(c("Median", "MAD", "Mean", "SD", "MAP", "Component") %in% colnames(x)))
expect_equal(nrow(x), 14)
#> Error: nrow(x) not equal to 14.
#> 1/1 mismatches
#> [1] 10 - 14 == -4

x <- eti(bfit)
expect_equal(nrow(x), 14)
#> Error: nrow(x) not equal to 14.
#> 1/1 mismatches
#> [1] 10 - 14 == -4

x <- hdi(bfit)
expect_equal(nrow(x), 14)
#> Error: nrow(x) not equal to 14.
#> 1/1 mismatches
#> [1] 10 - 14 == -4

x <- p_direction(bfit)
expect_equal(nrow(x), 14)
#> Error: nrow(x) not equal to 14.
#> 1/1 mismatches
#> [1] 10 - 14 == -4

x <- rope(bfit, range = c(-.1, .1))
expect_equal(nrow(x), 14)
#> Error: nrow(x) not equal to 14.
#> 1/1 mismatches
#> [1] 10 - 14 == -4

x <- p_rope(bfit, range = c(-.1, .1))
expect_equal(nrow(x), 14)
#> Error: nrow(x) not equal to 14.
#> 1/1 mismatches
#> [1] 10 - 14 == -4

x <- p_map(bfit)
expect_equal(nrow(x), 14)
#> Error: nrow(x) not equal to 14.
#> 1/1 mismatches
#> [1] 10 - 14 == -4

x <- p_significance(bfit, threshold = c(-.1, .1))
expect_equal(nrow(x), 14)
#> Error: nrow(x) not equal to 14.
#> 1/1 mismatches
#> [1] 10 - 14 == -4

x <- equivalence_test(bfit, range = c(-.1, .1))
expect_equal(nrow(x), 14)
#> Error: nrow(x) not equal to 14.
#> 1/1 mismatches
#> [1] 10 - 14 == -4

x <- estimate_density(bfit)
expect_equal(length(unique(x$Parameter)), 14)
#> Error: length(unique(x$Parameter)) not equal to 14.
#> 1/1 mismatches
#> [1] 10 - 14 == -4
```

<sup>Created on 2023-03-30 with [reprex v2.0.2](https://reprex.tidyverse.org)</sup>

<details style="margin-bottom:10px;">
<summary>
Session info
</summary>

``` r
sessioninfo::session_info()
#> ─ Session info ───────────────────────────────────────────────────────────────
#>  setting  value
#>  version  R version 4.2.3 (2023-03-15 ucrt)
#>  os       Windows 10 x64 (build 19044)
#>  system   x86_64, mingw32
#>  ui       RTerm
#>  language (EN)
#>  collate  English_Europe.utf8
#>  ctype    English_Europe.utf8
#>  tz       Europe/Paris
#>  date     2023-03-30
#>  pandoc   3.1 @ C:/Users/etienne/AppData/Local/Pandoc/ (via rmarkdown)
#> 
#> ─ Packages ───────────────────────────────────────────────────────────────────
#>  ! package        * version   date (UTC) lib source
#>    abind            1.4-5     2016-07-21 [1] CRAN (R 4.2.0)
#>    backports        1.4.1     2021-12-13 [1] CRAN (R 4.2.0)
#>    bayesplot        1.10.0    2022-11-16 [1] CRAN (R 4.2.2)
#>    bayestestR     * 0.13.0.10 2023-03-30 [1] local
#>    blavaan        * 0.4-7     2023-03-01 [1] CRAN (R 4.2.2)
#>    brio             1.1.3     2021-11-30 [1] CRAN (R 4.2.0)
#>    callr            3.7.3     2022-11-02 [1] CRAN (R 4.2.2)
#>    checkmate        2.1.0     2022-04-21 [1] CRAN (R 4.2.0)
#>    cli              3.6.0     2023-01-09 [1] CRAN (R 4.2.2)
#>    cmdstanr       * 0.5.3     2022-08-01 [1] local
#>    coda             0.19-4    2020-09-30 [1] CRAN (R 4.2.0)
#>    codetools        0.2-19    2023-02-01 [1] CRAN (R 4.2.2)
#>    colorspace       2.1-0     2023-01-23 [1] CRAN (R 4.2.2)
#>    CompQuadForm     1.4.3     2017-04-12 [1] CRAN (R 4.2.0)
#>    crayon           1.5.2     2022-09-29 [1] CRAN (R 4.2.1)
#>    curl             5.0.0     2023-01-12 [1] CRAN (R 4.2.2)
#>    datawizard       0.7.0.5   2023-03-27 [1] https://easystats.r-universe.dev (R 4.2.3)
#>    desc             1.4.2     2022-09-08 [1] CRAN (R 4.2.1)
#>    digest           0.6.31    2022-12-11 [1] CRAN (R 4.2.2)
#>    distributional   0.3.2     2023-03-22 [1] CRAN (R 4.2.3)
#>    dplyr            1.1.1     2023-03-22 [1] CRAN (R 4.2.3)
#>    evaluate         0.20      2023-01-17 [1] CRAN (R 4.2.2)
#>    fansi            1.0.4     2023-01-22 [1] CRAN (R 4.2.2)
#>    farver           2.1.1     2022-07-06 [1] CRAN (R 4.2.1)
#>    fastmap          1.1.1     2023-02-24 [1] CRAN (R 4.2.2)
#>    fs               1.6.0     2023-01-23 [1] CRAN (R 4.2.2)
#>    future           1.32.0    2023-03-07 [1] CRAN (R 4.2.2)
#>    future.apply     1.10.0    2022-11-05 [1] CRAN (R 4.2.2)
#>    generics         0.1.3     2022-07-05 [1] CRAN (R 4.2.1)
#>    ggplot2          3.4.1     2023-02-10 [1] CRAN (R 4.2.2)
#>    globals          0.16.2    2022-11-21 [1] CRAN (R 4.2.2)
#>    glue             1.6.2     2022-02-24 [1] CRAN (R 4.2.0)
#>    gridExtra        2.3       2017-09-09 [1] CRAN (R 4.2.0)
#>    gtable           0.3.3     2023-03-21 [1] CRAN (R 4.2.3)
#>    htmltools        0.5.5     2023-03-23 [1] CRAN (R 4.2.3)
#>    inline           0.3.19    2021-05-31 [1] CRAN (R 4.2.0)
#>    insight          0.19.1.2  2023-03-23 [1] https://easystats.r-universe.dev (R 4.2.3)
#>    jsonlite         1.8.4     2022-12-06 [1] CRAN (R 4.2.2)
#>    knitr            1.42      2023-01-25 [1] CRAN (R 4.2.2)
#>    lattice          0.20-45   2021-09-22 [2] CRAN (R 4.2.3)
#>    lavaan         * 0.6-15    2023-03-14 [1] CRAN (R 4.2.3)
#>    lifecycle        1.0.3     2022-10-07 [1] CRAN (R 4.2.1)
#>    listenv          0.9.0     2022-12-16 [1] CRAN (R 4.2.2)
#>    loo              2.5.1     2022-03-24 [1] CRAN (R 4.2.0)
#>    magrittr         2.0.3     2022-03-30 [1] CRAN (R 4.2.0)
#>    MASS             7.3-58.3  2023-03-07 [1] CRAN (R 4.2.2)
#>    Matrix           1.5-3     2022-11-11 [1] CRAN (R 4.2.2)
#>    matrixStats      0.63.0    2022-11-18 [1] CRAN (R 4.2.2)
#>    mnormt           2.1.1     2022-09-26 [1] CRAN (R 4.2.1)
#>    munsell          0.5.0     2018-06-12 [1] CRAN (R 4.2.0)
#>    mvtnorm          1.1-3     2021-10-08 [1] CRAN (R 4.2.0)
#>    nonnest2         0.5-5     2020-07-05 [1] CRAN (R 4.2.0)
#>    parallelly       1.35.0    2023-03-23 [1] CRAN (R 4.2.3)
#>    pbivnorm         0.6.0     2015-01-23 [1] CRAN (R 4.2.0)
#>    pillar           1.9.0     2023-03-22 [1] CRAN (R 4.2.3)
#>    pkgbuild         1.4.0     2022-11-27 [1] CRAN (R 4.2.2)
#>    pkgconfig        2.0.3     2019-09-22 [1] CRAN (R 4.2.0)
#>    pkgload          1.3.2     2022-11-16 [1] CRAN (R 4.2.2)
#>    posterior        1.4.1     2023-03-14 [1] CRAN (R 4.2.2)
#>    prettyunits      1.1.1     2020-01-24 [1] CRAN (R 4.2.0)
#>    processx         3.8.0     2022-10-26 [1] CRAN (R 4.2.2)
#>    ps               1.7.3     2023-03-21 [1] CRAN (R 4.2.3)
#>    purrr            1.0.1     2023-01-10 [1] CRAN (R 4.2.2)
#>    quadprog         1.5-8     2019-11-20 [1] CRAN (R 4.2.0)
#>    R.cache          0.16.0    2022-07-21 [1] CRAN (R 4.2.1)
#>    R.methodsS3      1.8.2     2022-06-13 [1] CRAN (R 4.2.0)
#>    R.oo             1.25.0    2022-06-12 [1] CRAN (R 4.2.0)
#>    R.utils          2.12.2    2022-11-11 [1] CRAN (R 4.2.1)
#>    R6               2.5.1     2021-08-19 [1] CRAN (R 4.2.0)
#>    Rcpp           * 1.0.10    2023-01-22 [1] CRAN (R 4.2.2)
#>  D RcppParallel     5.1.7     2023-02-27 [1] CRAN (R 4.2.2)
#>    reprex           2.0.2     2022-08-17 [1] CRAN (R 4.2.1)
#>    rlang            1.1.0     2023-03-14 [1] CRAN (R 4.2.3)
#>    rmarkdown        2.21      2023-03-26 [1] CRAN (R 4.2.3)
#>    rprojroot        2.0.3     2022-04-02 [1] CRAN (R 4.2.0)
#>    rstan          * 2.26.13   2022-06-25 [1] local
#>    rstantools       2.3.0     2023-03-09 [1] CRAN (R 4.2.2)
#>    rstudioapi       0.14      2022-08-22 [1] CRAN (R 4.2.1)
#>    sandwich         3.0-2     2022-06-15 [1] CRAN (R 4.2.0)
#>    scales           1.2.1     2022-08-20 [1] CRAN (R 4.2.1)
#>    sessioninfo      1.2.2     2021-12-06 [1] CRAN (R 4.2.0)
#>    StanHeaders    * 2.26.13   2022-06-25 [1] local
#>    styler           1.9.1     2023-03-04 [1] CRAN (R 4.2.2)
#>    tensorA          0.36.2    2020-11-19 [1] CRAN (R 4.2.0)
#>    testthat       * 3.1.7     2023-03-12 [1] CRAN (R 4.2.3)
#>    tibble           3.2.0     2023-03-08 [1] CRAN (R 4.2.2)
#>    tidyselect       1.2.0     2022-10-10 [1] CRAN (R 4.2.1)
#>    tmvnsim          1.0-2     2016-12-15 [1] CRAN (R 4.2.0)
#>    utf8             1.2.3     2023-01-31 [1] CRAN (R 4.2.2)
#>    V8               4.2.2     2022-11-03 [1] CRAN (R 4.2.2)
#>    vctrs            0.6.1     2023-03-22 [1] CRAN (R 4.2.3)
#>    waldo            0.4.0     2022-03-16 [1] CRAN (R 4.2.0)
#>    withr            2.5.0     2022-03-03 [1] CRAN (R 4.2.0)
#>    xfun             0.37      2023-01-31 [1] CRAN (R 4.2.2)
#>    yaml             2.3.7     2023-01-23 [1] CRAN (R 4.2.2)
#>    zoo              1.8-11    2022-09-17 [1] CRAN (R 4.2.1)
#> 
#>  [1] C:/Users/etienne/AppData/Local/R/win-library/4.2
#>  [2] C:/Users/etienne/AppData/Local/Programs/R/R-4.2.3/library
#> 
#>  D ── DLL MD5 mismatch, broken installation.
#> 
#> ──────────────────────────────────────────────────────────────────────────────
```

</details>

**Comment by IndrajeetPatil** [comment]
@strengejacke I think this is resolved in #625, right?

**Comment by strengejacke** [comment]
Yes, these should be fixed. Still, there's something not yet / no longer working, see https://github.com/easystats/bayestestR/issues/627.
